anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/forecast/interval/IntervalForecast.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.anomdetect.forecast.interval;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import static com.expedia.adaptivealerting.core.util.AssertUtil.isTrue;
+
+@Data
+@NoArgsConstructor
+public class IntervalForecast {
+    private double upperStrong;
+    private double upperWeak;
+    private double lowerWeak;
+    private double lowerStrong;
+
+    @JsonCreator
+    public IntervalForecast(
+            @JsonProperty("upperStrong") double upperStrong,
+            @JsonProperty("upperWeak") double upperWeak,
+            @JsonProperty("lowerWeak") double lowerWeak,
+            @JsonProperty("lowerStrong") double lowerStrong) {
```

Comment:
Do we need the `@JsonProperty` when the name is the same as the parameter?
Are these also always required?

Reply:
I'm not sure actually. In Spring you do not as long as debug info isn't disabled. In Jackson I don't know. I'll look it up.